pkgs/development/compilers/llvm/7/llvm.nix

```diff
@@ -13,7 +13,8 @@
 , debugVersion ? false
 , enableManpages ? false
 , enableSharedLibraries ? true
-, enableWasm ? true
+, targets ? [ stdenv.hostPlatform stdenv.targetPlatform ]
+, enableWasm ? true # TODO fold this into `targets` somehow
```

Comment:
Can we maybe it remove it for now and just enable it by default?

Reply:
Eh I'm fine with how it is now. If anything, this TODO is really for later LLVMs when WASM becomes a stable backend.